README.md

```diff
@@ -12,28 +12,185 @@ We believe that, like native platforms, the web deserves a first-class virtual s
 
 For more details on the motivation, see [Motivation.md](./Motivation.md).
 
-## Prototypes
+## Sample code
 
-We are not yet near the point of standardization, and are first working on proving out what the result should look like via JavaScript prototypes. So far this repository hosts two prototype implementations, each in separate branches:
+```html
+<!--
+  virtual-scroller lives in a built-in module and needs to be imported before use.
+  (The name of the module is subject to change.)
+-->
+<script type="module">
+import "std:virtual-scroller";
```

Comment:
Is the different name of module vs element intentional?

Reply:
I'm not sure :-/. Maybe we should just do `std:virtual-content` for now to keep them in sync, and figure out the final name (for both) later. That would mean our repository name, as well as the "concept" we talk about, is out of sync with the element name. But maybe that's not too bad.

Or, maybe we should just rename virtual-content to virtual-scroller, and use virtual-scroller everywhere for now, with the understanding that there's a final rename coming eventually. What do you think?